src/modules/projects/components/ProjectExternalFormSubmissions.tsx

```diff
@@ -0,0 +1,64 @@
+import { Paper } from '@mui/material';
+import React, { useState } from 'react';
+import Loading from '@/components/elements/Loading';
+import PageTitle from '@/components/layout/PageTitle';
+import useSafeParams from '@/hooks/useSafeParams';
+import FormSelect from '@/modules/form/components/FormSelect';
+import { getRequiredLabel } from '@/modules/form/components/RequiredLabel';
+import { usePickList } from '@/modules/form/hooks/usePickList';
+import { isPickListOption } from '@/modules/form/types';
+import { itemDefaults } from '@/modules/form/util/formUtil';
+import ProjectSubmissionsTable from '@/modules/projects/components/tables/ProjectSubmissionsTable';
+import { ItemType, PickListOption, PickListType } from '@/types/gqlTypes';
+
+interface Props {}
+const ProjectExternalFormSubmissions: React.FC<Props> = ({}) => {
+  const { projectId } = useSafeParams() as {
+    projectId: string;
+  };
+
+  const { pickList: formTypeList, loading: formTypeListLoading } = usePickList({
+    item: {
+      ...itemDefaults,
+      linkId: 'fake',
+      type: ItemType.Choice,
+      pickListReference: PickListType.ExternalFormTypesForProject,
+    },
+    projectId,
+  });
```

Comment:
This is fine.  I think it's a little cleaner to just use the query directly:
```
  const { data, loading, error } = useGetPickListQuery({
    variables: {
      pickListType: PickListType.ExternalFormTypesForProject,
      projectId,
    },
  })
```

Reply:
Thanks, this is much nicer!